### PR TITLE
add a RPC call to flush buffers and stop workers

### PR DESCRIPTION
**Pull request by tagomoris**
revised version of #1111.

### CODE REVIEW

File: lib/fluent/supervisor.rb [hunk 1]
```diff
@@ -82,6 +82,16 @@ def run_rpc_server
         end
         nil
       }
+      @rpc_server.mount_proc('/api/plugins.flushBuffersAndKillWorkers') { |req, res|
```
Comment (repeatedly) [inline lib/fluent/supervisor.rb]: killWorkers operation is not related with plugins so adding it to `plugins` prefix seems not good.
processes is better.

Comment (tagomoris) [inline lib/fluent/supervisor.rb]: Understood.

**Comment by tagomoris** [comment]
@repeatedly could you review once again?

File: lib/fluent/supervisor.rb [hunk 2]
```diff
@@ -75,6 +75,16 @@ def run_rpc_server
         Process.kill :TERM, $$
         nil
       }
+      @rpc_server.mount_proc('/api/processes.flushBuffersAndKillWorkers') { |req, res|
+        $log.debug "fluentd RPC got /api/plugins.flushBuffersAndKillWorkers request"
```
Comment (repeatedly) [inline lib/fluent/supervisor.rb]: plugins -> processes

Comment (tagomoris) [inline lib/fluent/supervisor.rb]: uguh, fixed :P

**Comment by repeatedly** [comment]
LGTM

**Pull request merged**